_posts/en/newsletters/2020-12-09-newsletter.md

```diff
@@ -0,0 +1,224 @@
+  don't implement case insensitive RFC3986 schema parsing.  Anyone
+  interested in addressing the problem is encouraged to [help
+  test][btcpay #2110] so that the developers of the affected wallets can
+  be notified about the issue.
+
+- **Cryptocurrency Open Patent Alliance (COPA) gains new members:**
+  Square Crypto [announced][square tweet] on Twitter that several new organizations
+  joined this alliance to prevent patents from being abused to prevent
+  innovation and adoption of cryptocurrency technology.  We join many
+  others in thanking all the current members of COPA: ARK.io, Bithyve,
+  Blockchain Commons, Blockstack, Blockstream, Carnes Validadas,
+  Cloudeya Ltd., Coinbase, Foundation Devices, Horizontal Systems,
+  Kraken, Mercury Cash, Protocol Labs, Request Network, SatoshiLabs,
+  Square, Transparent Systems, and VerifyChain.
+
+{% comment %}<!-- Here's me hacking my own policy that (1) releases
+should now be in the Releases section, (2) I only cover releases of the
+projects listed in the Notable Code Changes section, and (3) Minsc isn't
+one of those projects.  The following isn't technically a release
+announcement, it's the announcement new features which *just happens* to
+correspond with a release.  Yeah, maybe I should have better policies or
+just be more laid back.-->{% endcomment %}
+
+- **Minsc adds new features:** a recent [release][minsc 0.2] of this
+  policy language and compiler adds support for several new data types:
+  pubkeys, hashes, polices, [miniscript][topic miniscript],
+  [descriptors][topic descriptors], addresses, and networks.  It also
+  adds new functions for conversions: `miniscript()`, `wsh()`, `wpkh()`,
+  `sh()` and `address()`.  Here's an [example][minsc example] taken from
+  the project's website that uses a specified public key, as a [BIP32][]
+  extended public key with derivation path, and several conversion
+  functions, returning the set of data specified on the final line:
+
+    ```hack
+    $alice = xpub68Gmy5EdvgibQVfPdqkBBCHxA5htiqg55crXYuXoQRKfDBFA1WEjWgP6LHhwBZeNK1VTsfTFUHCdrfp1bgwQ9xv5ski8PX9rL2dZXvgGDnw/9/0;
+
+    $policy = pk($alice/1/3) && older(1 month);
+    $miniscript = miniscript($policy); // compile policy to miniscript
+    $descriptor = wsh($miniscript); // wrap with a p2wsh descriptor
+    $address = address($descriptor); // generate the address
+
+    [ $policy, $miniscript, $descriptor, $address ]
+    ```
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review Club][]
+meeting, highlighting some of the important questions and answers.  Click on a
+question below to see a summary of the answer from the meeting.*
+
+[Periodically make block-relay connections and sync headers][review club
+#19858] is a PR ([#19858][Bitcoin Core #19858]) by Suhas Daftuar to make [eclipse
+attacks][topic eclipse attacks] more difficult.  The proposal initiates
```

Comment:
Can we re-add s/more difficult/more difficult by increasing the existing outbound peer rotation/? ISTM this is an important point.

Reply:
@jonatack I'm sorry, I didn't know what "increasing the existing outbound peer rotation" meant.  (I still don't.)  It seemed to me that the next sentence describes what the proposal actually does.

Reply:
Oh, existing outbound peer rotation refers to our current logic of stale tip detection and replacement with a new full-relay connection (thus the original sentence of "by increasing the existing outbound peer rotation beyond the current logic of stale tip detection and replacement with a new full-relay connection"). It's hard for me to know if that is too much detail, but to me it was interesting as it is currently our sole outbound rotation mechanism.